### PR TITLE
Update install procedure

### PR DESCRIPTION
**Pull request by gbenhaim**
ci:+1:

### CODE REVIEW

File: docs/Install.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+That's It ! You have finished the installation process, now it's time to create some
+cool virtual environment :)
+
+**NOTE**: In order to enable nested virtualization check this
+`document <EnableNested>`_
+
+.. _repoman: http://repoman.readthedocs.io
+
+.. _ci-tools repo: http://resources.ovirt.org/repos/ci-tools
```
Comment (david-caro) [inline docs/Install.rst]: This is not used anywhere, you should add a note on why/how to add this repo

File: docs/oVirt_System_Tests.rst [hunk 2]
```diff
@@ -0,0 +1,463 @@
+Running oVirt System Tests
```
Comment (david-caro) [inline docs/oVirt_System_Tests.rst]: I think this should end up in the ovirt-system-tests repo eventually

**Comment by david-caro** [comment]
ci test please

File: docs/Install.rst [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+This will install the needed package.
+
+
+libvirt
+~~~~~~~~~
+
+Make sure libvirt is configured to run::
+
+    $ systemctl enable libvirtd
+     $ systemctl start libvirtd
```
Comment (david-caro) [inline docs/Install.rst]: extra space

File: docs/Install.rst [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+User setup
+~~~~~~~~~~~~~
+
+Running lago requires certain permissions, so the user running it should be
+part of certain groups.
+
+Add yourself to lago and qemu groups::
+
+    $ usermod -a -G lago USERNAME
+     $ usermod -a -G qemu USERNAME
```
Comment (david-caro) [inline docs/Install.rst]: extra space (or is it intentional?)

File: docs/Install.rst [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+rights, to make sure you can just run::
+
+    $ chmod g+x $HOME
+
+And, just to be sure, let's refresh libvirtd service to ensure that it
+refreshes it's permissions and picks up any newly created users::
+
+    $ sudo service libvirtd restart
+
+That's It ! You have finished the installation process, now it's time to create some
```
Comment (david-caro) [inline docs/Install.rst]: Extra space (before the `!`)

File: docs/index.rst [hunk 6]
```diff
@@ -9,7 +9,9 @@ Getting started
 .. toctree::
     :maxdepth: 3
 
-    README
+    Install
+    EnableNested
```
Comment (david-caro) [inline docs/index.rst]: Hmm, maybe we should stick to a naming scheme (camel-case, with `_`...)

**Comment by david-caro** [comment]
Can you rebase please? That will also regenerate the html on the jenkins jobs to check the formatting.

**Comment by mykaul** [comment]
@gbenhaim - is this still relevant with @eedri 's fixes?

**Comment by eedri** [comment]
@gbenhaim @nvgoldin I think we can abandon this, as most docs for OST moved into 
ovirt-system-tests.readthedocs.io and install info was refactored in Lago with https://github.com/lago-project/lago/pull/330

**Pull request closed without merging**